tests/integration/modifiers/scroll-into-view-test.js

```diff
@@ -97,4 +97,135 @@ module('Integration | Modifier | scroll-into-view', function (hooks) {
       'scrollIntoView was not called'
     );
   });
+
+  module('with offset', function (_hooks) {
+    _hooks.beforeEach(function () {
+      this.scrollToSpy = sandbox.spy(window, 'scrollTo');
```

Comment:
I know the test already uses `sandbox`, but it's a bad practice. You don't need to fix the existing tests if you don't want to, but let's not propagate this pattern. We already have the cleanup hooked up here:

https://github.com/elwayman02/ember-scroll-modifiers/blob/master/tests/test-helper.js#L13

You can just do `sinon.spy` and `sinon.stub` directly, and there's no need to create a `sandbox` variable at all or restore it in `afterEach`.